.docker/nginx.conf

```diff
@@ -0,0 +1,38 @@
+server {
+    listen 8080;
+    server_name localhost;
+    root /usr/share/nginx/html;
+
+    gzip on;
```

Comment:
isn't that a bit out of scope for a configuration built into a Docker image? The same way we don't implement HTTPS in this context but we terminate it in a load balancer or proxy

Reply:
Replaced with an `include` in https://github.com/libero/storybook/pull/4/commits/e75805749fc4a1751430e6e1c5eb8345053d7303.